### PR TITLE
hv: pci: rename CFG read/write function for PCI-compatible Configurat…

### PR DESCRIPTION
**Pull request by lifeix**
…ion Mechanism

Move CFG read/write function for PCI-compatible Configuration Mechanism from
debug/uartuart16550.c to hw/pci.c and rename CFG read/write function for
PCI-compatible Configuration Mechanism to pci_pio_read/write_cfg to align with
CFG read/write function pci_mmcfg_read/write_cfg for PCI Express Enhanced
Configuration Access Mechanism.

Tracked-On: #4371
Signed-off-by: Li Fei1 <fei1.li@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
6 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:common filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:75<br> reason:Static procedure is not explicitly called in code analysed. : pci_pio_read_cfg<br><2> config:part_config filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:80<br> reason:Function return value potentially unused. : addr<br><3> config:sharing_config filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:80<br> reason:Function return value potentially unused. : addr<br><4> config:common filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:101<br> reason:Static procedure is not explicitly called in code analysed. : pci_pio_write_cfg<br><5> config:part_config filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:103<br> reason:Function return value potentially unused. : addr<br><6> config:sharing_config filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:103<br> reason:Function return value potentially unused. : addr<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
6 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:common filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:75<br> reason:Static procedure is not explicitly called in code analysed. : pci_pio_read_cfg<br><2> config:part_config filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:80<br> reason:Function return value potentially unused. : addr<br><3> config:sharing_config filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:80<br> reason:Function return value potentially unused. : addr<br><4> config:common filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:101<br> reason:Static procedure is not explicitly called in code analysed. : pci_pio_write_cfg<br><5> config:part_config filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:103<br> reason:Function return value potentially unused. : addr<br><6> config:sharing_config filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:103<br> reason:Function return value potentially unused. : addr<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: hypervisor/hw/pci.c [hunk 1]
```diff
@@ -61,6 +61,81 @@ uint64_t get_mmcfg_base(void)
 	return pci_mmcfg_base;
 }
 
+#if defined(HV_DEBUG)
+static inline uint32_t pci_off_to_address(union pci_bdf bdf, uint32_t offset)
```
Comment (dongyaozu) [inline hypervisor/hw/pci.c]: pio_off_to_address?

Comment (dongyaozu) [reply hypervisor/hw/pci.c]: Also for pci_mmcfg_calc_address, please rename similarly?

Comment (lifeix) [reply hypervisor/hw/pci.c]: done

**Comment by HuiHuangShi** [comment]
6 new potential violations to the coding guideline detected. Please help check. Thanks!<br><1> config:common filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:75<br> reason:Static procedure is not explicitly called in code analysed. : pci_pio_read_cfg<br><2> config:part_config filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:80<br> reason:Function return value potentially unused. : addr<br><3> config:sharing_config filename:/hypervisor/hw/pci.c func:pci_pio_read_cfg offset:80<br> reason:Function return value potentially unused. : addr<br><4> config:common filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:101<br> reason:Static procedure is not explicitly called in code analysed. : pci_pio_write_cfg<br><5> config:part_config filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:103<br> reason:Function return value potentially unused. : addr<br><6> config:sharing_config filename:/hypervisor/hw/pci.c func:pci_pio_write_cfg offset:103<br> reason:Function return value potentially unused. : addr<br><br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**